WooCommerce/Classes/Authentication/Jetpack Setup/Native Jetpack Setup/LoginJetpackSetupViewModel.swift

```diff
@@ -225,14 +236,17 @@ private extension LoginJetpackSetupViewModel {
             case .success(let user):
                 guard let connectedEmail = user.wpcomUser?.email else {
                     DDLogWarn("⚠️ Cannot find connected WPcom user")
-                    return // TODO: add tracks and handle error
+                    self.analytics.track(.loginJetpackSetupCannotFindWPCOMUser)
+                    return // TODO: handle error
                 }
 
                 self.jetpackConnectedEmail = connectedEmail
                 self.currentConnectionStep = .authorized
                 self.currentSetupStep = .done
+
+                self.analytics.track(.loginJetpackSetupAllStepsMarkedDone)
             case .failure(let error):
-                // TODO: add tracks
+                self.analytics.track(.loginJetpackSetupErrorCheckingJetpackConnection)
```

Comment:
Since we are doing auto-retry, should we track this above when `retryCount` exceeds the limit instead?

Reply:
That is a great point. Done in https://github.com/woocommerce/woocommerce-ios/pull/8187/commits/f88c1b7e4c167618463bae02c866565a281801b2